examples/index.ts

```diff
@@ -39,9 +39,9 @@ const main = async () => {
   }
 
   console.log('Listing caches:');
-  let token;
+  let token: string | undefined;
```

Comment:
I don't have the javascript experience to have a well formed opinion here, but I'm seeing a lot of comments saying that explicitly initializing to `undefined` should be avoided because `undefined` means uninitialized.
Does that mean the `getNextToken()` method should return `null` instead of `undefined`? A question for a different pr perhaps.

Reply:
okay then this is fine how it is for now